ws-nextjs-app/pages/[service]/live/[id]/Header/LiveLabelHeader/index.tsx

```diff
@@ -21,20 +21,18 @@ const LiveLabelHeader = ({
 }: PropsWithChildren<LiveLabelPromoProps>) => {
   return (
     <div data-testid="live-label">
-      <LiveLabel.Pulse
-        className={className}
-        width="24"
-        height="24"
-        css={styles.liveLabelPulse}
-      />
       <LiveLabel.Text
         lang={lang}
         id={id}
         offScreenText={offScreenText}
-        css={
-          isHeaderImage
-            ? styles.liveLabelTextWithImage
-            : styles.liveLabelTextWithoutImage
+        isHeaderImage={isHeaderImage}
+        pulse={
+          <LiveLabel.Pulse
+            className={className}
+            width="24"
+            height="24"
+            css={styles.liveLabelPulse}
+          />
```

Comment:
I'm not 100% sure on this method of passing this component down as a prop, I wonder if we need to break down the LiveLabel.Text into smaller components to accommodate the change, keen on finding out what other people's opinions are on this

Reply:
This makes sense, will try breaking down the Text component further to avoid this - it did feel kind of wrong as i was doing it !